content/actions/reference/workflow-syntax-for-github-actions.md

```diff
@@ -642,6 +643,15 @@ steps:
     shell: pwsh
 ```
 
+#### Example running a script using PowerShell Desktop
```

Comment:
We're [currently revising](https://github.com/github/docs/issues/2063) how section titles are used in this guide, so this would be a future-proofed title to have here 👍 
```suggestion
#### Example: Using PowerShell Desktop to run a script
```